tests/e2e/unified_test.go

```diff
@@ -197,6 +197,17 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 						u := bytesToUnstructured(t, fixture.Update, uniqueID, project)
 						opt.Updates = append(opt.Updates, u)
 					}
+
+					// We want to use SSA everywhere, but some of our tests are broken by SSA
+					switch group := primaryResource.GetObjectKind().GroupVersionKind().Group; group {
```

Comment:
We can effectively filter by API groups only in the "fixtures" tests. For the samples test, maybe we just stick with "no SSA" for now.

Reply:
Good call, will set no-ssa for samples